pages/content.js

```diff
@@ -46,8 +163,10 @@ export default function Content({ currentCollections }) {
         )}
         <h3>{currentArticle.title}</h3>
         <StyledContent
+          className="articleFullContent"
```

Comment:
are you using ths className to find the Element for inserting your highlights, right? Here you should use the 'useRef' hook to get the reference to the element.

Reply:
ok thank you!